src/pages/home/_trade-markets.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import {
+    CardContainer,
+    CardHeader,
+    CardContent,
+    MarketsPartitionContainer,
+    StyledLogo,
+    TradeMarketsDesktop,
+    TradeMarketsMobile,
+} from './_home-style'
+import { Flex, SectionContainer, Carousel, WhiteText } from 'components/elements'
+import { localize } from 'components/localization'
+import CommoditiesLogo from 'images/svg/home/commodities.svg'
+import ForexLogo from 'images/svg/home/forex.svg'
+import SyntheticIndicesLogo from 'images/svg/home/synthetic-indices.svg'
+import StockIndicesLogo from 'images/svg/home/stock-indices.svg'
+
+const Card = ({ title, content, image }) => {
+    return (
+        <CardContainer>
+            <Flex m="auto">
+                <StyledLogo>
+                    <img src={image} alt={title} />
+                </StyledLogo>
+                <CardHeader>{title}</CardHeader>
+            </Flex>
+            <CardContent>{content}</CardContent>
+        </CardContainer>
+    )
+}
+
+export const TradeMarkets = () => {
+    const forex = {
+        title: 'Forex',
```

Comment:
<Localize

Reply:
Do I need to localize the title as well or only the content?